supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -144,18 +145,11 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
 
   def jesReceiveBehavior: Receive = LoggingReceive {
     case AbortJobCommand =>
-      serviceRegistryActor ! KvGet(ScopedKey(jobDescriptor.workflowDescriptor.id,
-        KvJobKey(jobDescriptor.key.call.fullyQualifiedName, jobDescriptor.key.index, jobDescriptor.key.attempt),
-        JesOperationIdKey))
-    case KvPair(scopedKey, operationId) if scopedKey.key == JesOperationIdKey =>
-      operationId match {
-        case Some(id) =>
-          jobLogger.info(s"$tag Aborting $id")
-          Try(Run(id, initializationData.genomics).abort()) match {
-            case Success(_) => jobLogger.info(s"$tag Aborted $id")
-            case Failure(ex) => jobLogger.warn(s"$tag Failed to abort $id: ${ex.getMessage}")
-          }
-        case None => jobLogger.warn(s"$tag Failed to abort ${jobDescriptor.call.fullyQualifiedName}. Run Id metadata field was empty.")
+      runId foreach { id =>
```

Comment:
Was it a deliberate choice to make the receive block synchronous against the remote service?